flink-ml-lib/src/test/java/org/apache/flink/ml/feature/VectorAssemblerTest.java

```diff
@@ -45,13 +45,16 @@
 
```

Comment:
Let's further add test cases to verify different handleInvalid strategies when the input vectors are valid (no null or NaN values), but the size of the vectors does not match with that set in `inputSizes`.

Reply:
I have add this case. If input vectors are valid (no null or NaN values), but the size of the vectors does not match with that set in inputSizes, the algorithm will throw Exception all the time. Because this err maybe data error or parameter error, I don't know which one is.

Reply:
I have refine code as spark's functions.